### ISSUE TITLE
Use more accounts how?

### ISSUE THREAD

**Issue by stickmick12** [mfron/PokemonGo-Bot#94]
i want to use all my 5 accounts at the same time but how i dont know. i am using visual studios 2015

**Comment by jiemojiemo** [comment]
You can copy Release to 5 different folders, then modify the config file for 5 accounts

**Comment by stickmick12** [comment]
and how i can copy it in virsual studios

**Comment by jiemojiemo** [comment]
You compile the code, and the Release will be created in the output directory.
exactly, H:\PokemonGo\Pokemon-Go-Bot\PokemonGo\RocketAPI\Window\bin\Release is in my computer

**Comment by stickmick12** [comment]
men i do not understand englisch so gut  come from germany

**Comment by jiemojiemo** [comment]
Ok, Simple word. Just copy it to diffirent folders

**Comment by mfron** [comment]
You can launch 5 time the .exe, no need to copie the folder

**Comment by stickmick12** [comment]
yes but then i must alltime if i start do again the settings and the login etc

**Comment by mfron** [comment]
yes, so duplicate the folder ^^

**Comment by stickmick12** [comment]
must i open it in virsual ?

**Comment by jiemojiemo** [comment]
Of Course not, find the output directory, the .exe is there

**Comment by stickmick12** [comment]
ahh i think i must it alltime open it in virsual. and why i must install virsual ?

**Comment by mfron** [comment]
You can install vs and build the project, or juste use the .exe on the release

**Comment by jiemojiemo** [comment]
For compile code...
Sometimes you can modify code for your features

**Comment by stickmick12** [comment]
ahhh for me oke thx also virsual etc is if i want any feature for me ?. and wenn i dont need any future i use the Build oke thx.
